src/Migration.php

```diff
@@ -1472,42 +1441,27 @@ protected function outputMessage($msg, $style = null, $area_id = null)
      * Output a message in console output.
      *
      * @param string $msg    Message to output.
-     * @param string $style  Style to use, see self::outputMessage() for possible values.
+     * @param ?string $style  Style to use, see {@link self::outputMessage()} for possible values.
      *
      * @return void
      */
-    private function outputMessageToCli($msg, $style = null)
+    private function outputMessageToCli(string $msg, ?string $style = null): void
     {
-
-        $format = null;
-        $verbosity = OutputInterface::VERBOSITY_NORMAL;
-        switch ($style) {
-            case 'title':
-                $msg       = str_pad(" $msg ", 100, '=', STR_PAD_BOTH);
-                $format    = 'info';
-                $verbosity = OutputInterface::VERBOSITY_NORMAL;
-                break;
-            case 'warning':
-                $msg       = str_pad("** {$msg}", 100);
-                $format    = 'comment';
-                $verbosity = OutputInterface::VERBOSITY_NORMAL;
-                break;
-            case 'strong':
-                $msg       = str_pad($msg, 100);
-                $format    = 'comment';
-                $verbosity = OutputInterface::VERBOSITY_NORMAL;
-                break;
-            case 'error':
-                $msg       = str_pad("!! {$msg}", 100);
-                $format    = 'error';
-                $verbosity = OutputInterface::VERBOSITY_QUIET;
-                break;
-            default:
-                $msg       = str_pad($msg, 100);
-                $format    = 'comment';
-                $verbosity = OutputInterface::VERBOSITY_VERBOSE;
-                break;
-        }
+        $msg = match ($style) {
+            'title' => str_pad(" $msg ", 100, '=', STR_PAD_BOTH),
+            'warning' => str_pad("** {$msg}", 100),
+            'error' => str_pad("!! {$msg}", 100),
+            default => str_pad($msg, 100),
+        };
+        $format = match ($style) {
+            'title' => 'info',
+            'error' => 'error',
+            default => 'comment',
+        };
+        $verbosity = match ($style) {
+            'error' => OutputInterface::VERBOSITY_QUIET,
```

Comment:
you choose to remove cases with VERBOSITY_NORMAL ? (at least move them to verbose)

Reply:
Wasn't intentional. Fixed now.